discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -239,6 +239,15 @@ def set_proxy_values(self, http_proxy, https_proxy='', no_proxy=''):
     def start_install(self):
         self.api_client.install_cluster(cluster_id=self.id)
 
+    def wait_for_logs_complete(self, timeout, interval=60, host_only=False):
```

Comment:
it's not clear what ``host_only`` means in this context

Reply:
just check the host logs to save waiting for timeout

Reply:
replaced to check_host_logs-only